.github/workflows/buildwoff2.yml

```diff
@@ -0,0 +1,122 @@
+name: Build woff2 font file
+
+on:
+  pull_request:
+    types: [synchronize, opened]
+
+permissions:
+  actions: read
+  checks: none
+  contents: write
+  deployments: none
+  issues: write
+  packages: none
+  pull-requests: write
+  repository-projects: none
+  security-events: none
+  statuses: none
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Detect branch for PR
+        id: vars
+        run: |
+          PR="${{ github.event.pull_request.number }}"
+
+          PR_INFO=$( curl \
+                   --request GET \
+                   --header 'authorization: Bearer ${{ secrets.GITHUB_TOKEN }}' \
+                   --header 'content-type: application/json' \
+                   --url https://api.github.com/repos/$GITHUB_REPOSITORY/pulls/$PR )
+          REF=$(echo "${PR_INFO}" | jq -r .head.ref)
+          BASE=$(echo "${PR_INFO}" | jq -r .head.repo.full_name)
+          STATE=$(echo "${PR_INFO}" | jq -r .state)
+          BASE_SHA=$(echo "${PR_INFO}" | jq -r .base.sha)
+
+          if [[ $STATE == "closed" ]]
+          then
+            echo "Pull Request already closed."
+            exit 0;
+          fi
+
+          echo ::set-output name=islocalbranch::$BASE == $GITHUB_REPOSITORY
+          echo ::set-output name=branch::$REF
+          echo ::set-output name=base::$BASE_SHA
```

Comment:
The detailed action log is showing a deprecation warning for the set-output command: 
```
Warning: The `set-output` command is deprecated 
and will be disabled soon. Please upgrade to using 
Environment Files. 
```
For more information see: 
https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/

`set-output` will fail after June 1st 2023, it looks straight forward to update to the new syntax, so maybe it would be good to do that now?

Reply:
Yes I know. But I didn't have time to check what exactly would need to be changed for that.
We will need to update that for all our actions, so will merge this as it is for now and we can adjust all our actions then at once for the new syntax. I'll create an issue for that.